internal/kibana/alerting.go

```diff
@@ -235,11 +263,38 @@ func getActionsFromResourceData(d *schema.ResourceData) ([]models.AlertingRuleAc
 				return []models.AlertingRuleAction{}, diag.FromErr(err)
 			}
 
-			actions = append(actions, models.AlertingRuleAction{
+			a := models.AlertingRuleAction{
 				Group:  action["group"].(string),
 				ID:     action["id"].(string),
 				Params: params,
-			})
+			}
+
+			currentAction := fmt.Sprintf("actions.%d", i)
+
+			if _, ok := d.GetOk(currentAction + ".frequency"); ok {
+				if serverVersion.LessThan(alertDelayMinSupportedVersion) {
```

Comment:
It seems that the `frequency` was added in [8.6+](https://github.com/elastic/kibana/pull/144130).

Reply:
Good catch, I added this in the beginning and forgot to update 🙏